lib/github.js

```diff
@@ -136,3 +136,24 @@ module.exports.getCommitStatusesForRef = async function getCommitStatusesForRef
     ref: branch
   })
 }
+
+module.exports.createPR = async function createPR (owner, repo, title, head, base, draft, body) {
+  return octokit.pulls.create({
+    owner,
+    repo,
+    title,
+    head,
+    base,
+    draft,
+    body
+  })
+}
+
+module.exports.updatePRStatus = async function updatePRStatus (owner, repo, pullNumber) {
```

Comment:
This doesn't allow you to set arbitrary state, so maybe a name change is in order?

```suggestion
module.exports.closePr = async function closePr (owner, repo, pullNumber) {
```

Reply:
👍